src/runtime/mod.rs

```diff
@@ -105,9 +90,17 @@ impl Runtime {
             }
         };
 
-        tokio::pin!(future);
+        local.spawn_local(drive);
 
-        self.local.spawn_local(drive);
+        Ok(Runtime { local, rt })
+    }
+
+    /// Runs a future to completion on the current runtime
+    pub fn block_on<F>(&self, future: F) -> F::Output
+    where
+        F: Future,
+    {
+        tokio::pin!(future);
```

Comment:
Do you mean to leave the commented code at line 107?

Reply:
I forgot that was there lol. We can delete this later in a cleanup PR.